assets.js

```diff
@@ -0,0 +1,14 @@
+//const CSS = [
+
+//];
```

Comment:
I'd remove this commented out section.

Reply:
https://github.com/jellyfin/jellyfin-web/pull/549/commits/651a53d64a2d44abaa7bfd9bf1bf58e5a7fe4122